erpnext/controllers/accounts_controller.py

```diff
@@ -1450,6 +1450,7 @@ def validate_and_delete_children(parent, data):
 	for d in deleted_children:
 		update_bin_on_delete(d, parent.doctype)
 
+
```

Comment:
if you drop this commit, semgrep won't crib about issues in that file.


Anyway, the problem is this line:

https://github.com/frappe/erpnext/blob/b0e77cbd89394a609108768466fef97cb00902bf/erpnext/controllers/accounts_controller.py#L228

This should be 
`data_field = None`

Reply:
Thanks for the suggestion, I added a fix for it anyway